components/import-action/import-action.composition.tsx

```diff
@@ -7,8 +7,10 @@ export const ImportActionExample = () => {
     <ThemeContext>
       <div style={styles}>
         <ImportAction
```

Comment:
consider passing bitId directly and extracting relevant data from that

Reply:
I will need to import `componentModel` to create the composition. its pretty extreme, no?